### DISCUSSION TITLE
SPAdes pipeline finished WITH WARNINGS! (spades 3.10.1)

### DISCUSSION THREAD

**Discussion by memaMJ** [ablab/spades#753] [Q&A]
SPAdes pipeline finished WITH WARNINGS!

=== Error correction and assembling warnings:
 * 0:12:38.703   544M / 2G    WARN    General                 (pair_info_count.cpp       : 140)   Too much reads aligned with negative insert size. Is the library orientation set properly?
======= Warnings saved to /Users/layel-5/Desktop/comp/spades_assembly/warnings.log

SPAdes log can be found here: /Users/layel-5/Desktop/comp/spades_assembly/spades.log

Thank you for using SPAdes!

so what dose that mean and how to fix it... 
am new to programming so pleas explain with some details

**Comment by asl** [comment]
So, what is your question?

**Comment by asl** [comment]
Well, the log reads:
```
Too much reads aligned with negative insert size. Is the library orientation set properly?
```
So, you need to check your input data, whether you specified it properly, whether the library preparation was ok, etc. It has nothing to do with programming.

**Comment by memaMJ** [comment]
I think the input data should be fine since it is a tutorial data for beginners...... am I wrong ?
what my next step should be ( in this case)  ?

**Comment by asl** [comment]
Given the warning, the data is probably not fine. Or you're specifying it improperly.
